app/apps/server/bridges/uploads.js

```diff
@@ -30,20 +30,31 @@ export class AppUploadBridge {
 		});
 	}
 
-	createUpload(details, buffer, appId) {
+	async createUpload(details = {}, buffer, appId) {
 		this.orch.debugLog(`The App ${ appId } is creating an upload "${ details.name }"`);
 
+		if (!details.userId) {
+			throw new Error('Missing user to perform the upload operation');
+		}
+
```

Comment:
This is not necessary, as the `userId` info will not be available when the upload is made by a livechat visitor